requirements/AUR-CBI-INT-0070/standalone-project/Makefile

```diff
@@ -0,0 +1,29 @@
+export CC = gcc
+export CFLAGS = -O0 -g -fno-omit-frame-pointer
+export LIBS =
+export LDFLAGS =
+
+export OBJDIR = "$(shell pwd)/objs"
+export LIBDIR = "$(shell pwd)/libs"
+export BINDIR = "$(shell pwd)/bin"
+
+SUBDIRS = src
+SUBDIRS_CLEAN = $(SUBDIRS%:=% clean)
+
+all: dirs $(SUBDIRS)
```

Comment:
The requirement is to be validated by a TEST, so I am missing a test script. It would be nice, if it could be invoked via make test.

Reply:
Done, it works